docs/bridged_networks.md

```diff
@@ -0,0 +1,108 @@
+A wishlist of IRC networks to be bridged is being collected [in a github issue](https://github.com/matrix-org/matrix-appservice-irc/issues/208).
+
+<table>
+    <tr>
+        <th>Network Name</th>
+        <th>Room alias format</th>
+        <th>Appservice user</th>
+        <th>Room for Support</th>
+        <th>Operator</th>
+    </tr>
+    <tr>
+        <td>darkfasel</td>
+        <td>#channame:darkfasel.net</td>
+        <td>@IRC-Darkfasel:darkfasel.net</td>
+        <td>#darkfasel:darkfasel.net</td>
```

Comment:
For these entries, could we use matrix.to links?

Reply:
now using matrix.to for all channels and appservice users